pkg/cmd/initialize/sigs.go

```diff
@@ -44,6 +44,8 @@ func CreateEventsFromSignatures(startId events.ID, sigs []detect.Signature) map[
 			evtDependency = append(evtDependency, eventDefID)
 		}
 
+		tags := append([]string{"signatures", "default"}, m.Tags...)
```

Comment:
Suggesting to treat it as a "set" in case "signatures" / "default" will be also mentioned in signature's Tags.

Reply:
Not sure I understood, you mean the naming?

Reply:
I think he meant to avoid duplicates.

Reply:
@josedonizetti you can use our own existing set package just to deduplicate then use the internal array.

Reply:
Correct. Prevent duplications.

Reply:
Sets are going to be renamed as tags. This is intentional, and part of the move to "everything is an event". In case of duplication, we can simply keep one tag and remove the duplication. In the future, signatures will be defined like regular (derived) events, and the signature engine will be removed, then it will also become more natural

Reply:
@yanivagman We're referring to the set data structure (in the context of deduplicating the tags slice), not the old name for tags.